### PR TITLE
doc: fix error information in a blog and remove redundant content

### PR DESCRIPTION
**Pull request by FeynmanZhou**
fix error information in a blog and remove redundant content as discussed in https://github.com/oras-project/oras-www/pull/147#discussion_r1179171771

### CODE REVIEW

**Review by TerryHowe — DISMISSED** [review]
LGTM

File: blog/authors.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ fzhou:
 
 yzha:
   name: Yi Zha
-  title:  Engineer, Microsoft
+  title:  ORAS Maintainer
```
Comment (shizhMSFT) [inline blog/authors.yml:20]: Currently, there is no public record that Yi is a maintainer of any repo in `oras-project`.

Comment (FeynmanZhou) [reply blog/authors.yml]: @shizhMSFT Thanks. I updated Yi's title.

File: blog/authors.yml [hunk 2]
```diff
@@ -3,21 +3,21 @@ oras:
...
   name: Feynman Zhou
   title: ORAS Maintainer
   url: https://github.com/FeynmanZhou
   image_url: https://github.com/FeynmanZhou.png
 
-yzha:
+yizha:
   name: Yi Zha
-  title:  Engineer, Microsoft
+  title:  ORAS Contributor, ACR Product Manager
```
Comment (shizhMSFT) [inline blog/authors.yml:20]: ACR is ambiguous. It is better to have the full name or within the Microsoft Azure namespace.

Comment (FeynmanZhou) [reply blog/authors.yml]: Updated to Microsoft Azure Product Manager

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**